### ISSUE TITLE
commitlint/config-angular no longer allows "chore"

### ISSUE THREAD

**Issue by ianwremmel** [greenkeeperio/greenkeeper#555]
Angular (and therefore [@commitlint/config-angular](https://www.npmjs.com/package/@commitlint/config-angular)) changed their allowed scopes, removing "chore" in favor of "ci" and "build". 

Since I lint my commit messages on CI, all my greenkeeper PRs are failing. Obviously, I can fix this on my side by changing the config of all my projects, but I imagine I'm not the only one blindly following the angular commit guidelines and linting my commits.

(To be clear, I don't have any good ideas for what you might do to deal with this, but I thought it worth bringing to your attention)

**Comment by gr2m** [comment]
what would be a good prefix for bumping a dev dependency in package.json? I use `chore:` for all kind of things, like edits on `.gitignore`, `LICENSE` or `.travis.yml`. I guess the latter could use "ci", but what about the others?

For reference, this is the commit introducing the change to angular: https://github.com/angular/angular/commit/dff6ee3

**Comment by ianwremmel** [comment]
Yea, I'm not really sure. I *think* if I'm reading the angular commit correctly:
- dependencies should be `build(npm)` 
- `.travis.yml` should be `ci(travis)`
- `.gitignore` is probably supposed to be `build(git)`
- `LICENSE` would maybe be `docs(license)` or `docs(legal)`?

These are all guesses and I feel like they replaced chore with two fields that don't cover everything chore used to. While changing greenkeeper to `build(npm)` seems like it would fix *my* issue, I'm guessing that'll break anyone who doesn't upgrade their commitlint rules. Though, I guess that's the point of greenkeeper... :)

**Comment by gr2m** [comment]
yeah I wish they would have deprecated it for at least one major version :) Let’s see what  Greenkeeper team thinks, I’m just helping out

**Comment by pvdlg** [comment]
I think https://conventionalcommits.org/ has been created in order to have a specification that still allow `chore`.

**Comment by Arcanemagus** [comment]
I believe that https://github.com/greenkeeperio/greenkeeper/pull/612 allows this issue to be closed as the message is now customizable if so desired.

**Comment by ianwremmel** [comment]
Seems reasonable to me. I'm not presently using greenkeeper, so I'm no longer in a position to confirm.
